tests/bugzilla_test.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+
+
+import json
+import uuid
+
+import loginsightwebhookdemo
+import loginsightwebhookdemo.bugzilla
+
+
+client = loginsightwebhookdemo.app.test_client()
+
+# Cannot add AlertName as must be unique for testing
+payload_base = {
+    "messages": [
+        {
+            "text": "hello world 1",
+            "timestamp": 1451940578545,
+            "fields": []
+        }
+    ]
+}
+
+URL = 'https://landfill.bugzilla.org/bugzilla-5.0-branch/'
+TOKEN = 'BR8zMD8ywOJSAgRnjylr4G3pa7Zx5zKE6U3xYl76'
```

Comment:
I opted for yes -- this is how it tests with an external server. I understand someone could change the credentials, however anyone can register for an account on this server today so I am hoping the risk is low and if problems arise, the fix is trivial.